src/module/classes/cleric.mjs

```
@@ -1,23 +1,43 @@
+ *   @param {object} activity - The activity performed.
+ *   @param {object} usageConfig - The usage configuration.
  */
-export function healOver(actor,combatants){    
-    console.log("Nothing ")
+export function infuseHeal(activity, usageConfig) {
+    const actor = activity.actor;
+    if (activity.type === "heal" && actor.items.find(i => i.name === "Infused Healer")) {
+        const spellLevel = Number.parseInt(usageConfig.spell.slot.replace('spell', ''));
+        const infusedValue = 2 + spellLevel;
+        if (actor.system.attributes.hp.value !== actor.system.attributes.hp.max) {
+            let newHpValue = Math.min(actor.system.attributes.hp.value + infusedValue, actor.system.attributes.hp.max)
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'newHpValue' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)